src/homelogstore/log_store.cpp

```diff
@@ -388,6 +391,16 @@ sisl::status_response HomeLogStore::get_status(const sisl::status_request& reque
     response.json["truncation_parallel_to_writes?"] = m_safe_truncation_boundary.active_writes_not_part_of_truncation;
     response.json["logstore_records"] = m_records.get_status(request.verbose_level);
     response.json["logstore_sb_first_lsn"] = m_logdev.m_logdev_meta.store_superblk(m_store_id).m_first_seq_num;
+
+    if (request.json.contains("type") && request.json["type"] == "logstore_record") {
+        log_dump_req dump_req{};
+        if (!request.next_cursor.empty()) { dump_req.start_seq_num = std::stoul(request.next_cursor); }
+        dump_req.end_seq_num = dump_req.start_seq_num + request.batch_size;
+        homestore::log_dump_verbosity verbose_level = homestore::log_dump_verbosity::HEADER;
+        if (request.json.contains("log_content")) { verbose_level = homestore::log_dump_verbosity::CONTENT; }
+        dump_req.verbosity_level = verbose_level;
+        response.json.update(dump_log_store(dump_req));
```

Comment:
Shouldn't dump_log_store consume dump_req.next_cursor to only do query from this next_cursor and batch_size? I didn't see it is being consume though. Is it expected?

Reply:
We fill the dump_req using the request.next_cursor at line 397.